packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import ChevronRightStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-right-stroke.svg';
+
+const LTR_COMPONENT_ID = 'breadcrumbs.chevron_right_stroke_icon';
+const RTL_COMPONENT_ID = 'breadcrumbs.chevron_left_stroke_icon';
+
+const getGardenId = (rtl: boolean) => (rtl ? RTL_COMPONENT_ID : LTR_COMPONENT_ID);
+
+/**
+ * Accepts all `<svg>` props
+ */
+export const StyledChevronIcon = styled(ChevronRightStrokeIcon).attrs(props => ({
+  'data-garden-id': `${getGardenId(props.theme.rtl)}`,
+  'data-garden-version': PACKAGE_VERSION,
+  'aria-hidden': 'true'
+}))`
+  ${props => props.theme.rtl && `transform:rotate(180deg);`}
```

Comment:
To match the other components use

```tsx
transform: ${props => props.theme.rtl && 'rotate(180deg)'};
```

Reply:
Whoops — thanks for pointing this out.  I will make sure to stay consistent with other styled-components CSS in the future.